tools/headless_testing/springsettings.cfg

```diff
@@ -1 +1,3 @@
 RapidTagResolutionOrder = repos-cdn.beyondallreason.dev;repos.beyondallreason.dev
+LogFlushLevel = 0
+ui_rendertotexture = 0
```

Comment:
shouldn't be needed in headless (if some widgets try to render to texture even in headless, they should probably be fixed not to do that)

Reply:
Removed `ui_rendertotexture = 0`

Tests still pass.